templates/icmpdef.j2

```diff
@@ -0,0 +1,5 @@
+{# ICMP Ping is dropped by default, unless we permit it per: #}
```

Comment:
I don't think we'd want to add this to the role.  

To allow icmp, we need to create a variable side "rule" to allow.